extension/intents/self/self.toml

```diff
@@ -113,3 +113,13 @@ match = """
 
 [[self.simpleTest.example]]
 phrase = "test"
+
+[self.helpIntent]
+description = "This will try to find help for for the intent on lexicon page"
+match = """
+  (show | open |) help (for | on |) [query] (to me | for me |)
+  (show | open |) command (for | on |) [query] (to me | for me |)
```

Comment:
I think this would be a better phrase:

```
(show | open |) (help | command{s}) (for | on | about |) [query] (to | for |) (me |)
```

It should match a few other forms of the phrase.